frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -0,0 +1,179 @@
+import { detailsPage } from '../../../../../packages/integration-tests-cypress/views/details-page';
+import { nav } from '../../../../../packages/integration-tests-cypress/views/nav';
+import { devNavigationMenu, switchPerspective } from '../constants/global';
+
+export const app = {
+  waitForLoad:(timeout: number = 30000) => {
+    cy.get('.co-m-loader', {timeout: timeout}).should('not.be.visible');
+  },
+  login:() => {
+    cy.get('body').then(($body) => {
+      if ($body.find('a[title="Log in with kube:admin"]').length) {
+        cy.get('a[title="Log in with kube:admin"]').click();
+          cy.url().should('include', 'login');
+      }
+    })
+    cy.get('#inputUsername').type(Cypress.env('username'));
+    cy.get('#inputPassword').type(Cypress.env('password'));
+    cy.get('[type="submit"]').click();
+    cy.get('[aria-label="Help menu"]').should('be.visible');
+  },
+}
+
+export const perspective = {
+  verifyPerspective: (perspectiveName: string) => {
+    cy.byLegacyTestID('perspective-switcher-toggle').should('contain.text', perspectiveName);
+  },
+
+  switchTo : (perspectiveName: switchPerspective) => {
```

Comment:
I tried to use enums as much as possible to make the coding easier, so thought to use this function

Reply:
If you feel this is a better approach, then I recommend updating the already existing helper methods/functions to use enums.  -thanks

Reply:
But that would affect the existing console automation code, isn't it?

Reply:
If you feel enums make the coding easier, then why not make it the standard.   Oh, are you saying this would require changing a lot of the console cypress tests?

Reply:
Still think you could update `nav.sidenav.switcher.changePerspectiveTo("Developer")` to use an enum: `nav.sidenav.switcher.changePerspectiveTo(switchPerspective.Developer)`.  I think you may have to only update one or two console cypress tests, we haven't used the `nav.sidenav.switcher` much yet.

Reply:
I'm ok if you want to address this in a follow up PR.